domain/src/main/java/com/globant/domain/usecase/implementation/GetHousesUseCaseImpl.kt

```diff
@@ -18,7 +18,7 @@ class GetHousesUseCaseImpl(
                 result
             }
             is Result.Failure -> {
-                database.getHouses()
+                result
```

Comment:
are you sure of this? if the service fails, this may call the database I think

Reply:
Yes I am sure, because I get the houses from database in other use case, in this one I just need to get the houses from the service and update the houses in database.